pkg/amber/endorsement.go

```diff
@@ -39,8 +39,12 @@ type VerifiedProvenanceSet struct {
 	Provenances []ProvenanceData
 }
 
-// ProvenanceData contains metadata about a provenance statement, identified by a URI and the
-// SHA256 digest of the content of the provenance.
+// ProvenanceData contains metadata about a provenance statement. The statement may be wrapped in a
+// DSSE envelope, or a Sigstore Bundle. The metadata identifies the provenance via a URI and a
```

Comment:
Understanding question: Does "the metadata identifie" means "The metadata consists of" or "contains"?

Reply:
Hmmm... Don't "consists of" and "contains" mean the same thing?

Reply:
Yes, I think "consist of" and "contains" mean the same roughly. 

I wasn't sure how the metadata relate to the the URI and SHA256 digest. 

What do you mean by "metadata identifies the provenance via a URI and SHA256 digest"? Are URI and SHA256 the metadata?

But this is nit :) We can follow up offline!

Reply:
Updated the text and removed "metadata".

Reply:
Nice, thanks, I get it now :)